Explorer/Assets/Scripts/SceneRunner/Scene/ISceneData.cs

```diff
@@ -59,6 +59,8 @@ public interface ISceneData
 
         bool IsSdk7();
 
+        bool IsPortableExperience();
```

Comment:
Im wondering if it would be better to convert it to a readonly property

Reply:
Well in this case I was following the same done for the IsSdk7, which in the implementation do this:
```
        public bool IsPortableExperience() =>
            SceneEntityDefinition.metadata.isPortableExperience;

        public bool IsSdk7() =>
            SceneEntityDefinition.metadata.runtimeVersion == "7";

```
But both could be converted into readonly properties tbh considering the metadata doesnt change...